contracts/mosaic/MerklePatriciaProof.sol

```diff
@@ -1,15 +1,11 @@
-/*
```

Comment:
Let's keep the original author versioning intact; we can reference that it is based on, or derived from ... (with a link)

Reply:
also, I believe these files were MIT, so we should keep such derived files explicitly MIT licensed in the header